### PR TITLE
New section for Error Codes

### PR DESCRIPTION
**Pull request by domaincomponents**
New section for Error Codes under Axon Server - Administration

### CODE REVIEW

File: axon-server/administration/error-codes.md [hunk 1]
```diff
@@ -0,0 +1,658 @@
...
+    </tr>
+    <tr>
+      <td style="text-align:left"></td>
+      <td style="text-align:left">AXONIQ-1000</td>
+      <td style="text-align:left">
+        <p><b>AUTHENTICATION_TOKEN_MISSING</b>
+        </p>
+        <p>This indicates that the Axon Server has been configured with authentication
+          enabled and will reject any instructions from a client application if the
+          token is missing.</p>
```
Comment (bert-laverman) [inline axon-server/administration/error-codes.md:28]: "...and a client application did not pass a token."

File: axon-server/administration/error-codes.md [hunk 2]
```diff
@@ -0,0 +1,658 @@
...
+    </tr>
+    <tr>
+      <td style="text-align:left"></td>
+      <td style="text-align:left">AXONIQ-1001</td>
+      <td style="text-align:left">
+        <p><b>AUTHENTICATION_INVALID_TOKEN</b>
+        </p>
+        <p>This indicates that the Axon Server has been configured with authentication
+          enabled and will reject any instructions from a client application if an
+          invalid token has been supplied.</p>
```
Comment (bert-laverman) [inline axon-server/administration/error-codes.md:39]: "...and a client application has supplied an invalid token."

File: axon-server/administration/error-codes.md [hunk 3]
```diff
@@ -0,0 +1,658 @@
...
+      </td>
+    </tr>
+    <tr>
+      <td style="text-align:left"></td>
+      <td style="text-align:left">AXONIQ-1004</td>
+      <td style="text-align:left">
+        <p><b>INSTRUCTION_RESULT_TIMEOUT</b>
+        </p>
+        <p>This indicates that the Axon Server threw an error during the execution
+          of the instruction passed to it.</p>
```
Comment (bert-laverman) [inline axon-server/administration/error-codes.md:69]: "This indicates that the execution of the instruction did not finish within the timeout set." I do not know if we have a property for this timeout.

File: axon-server/administration/error-codes.md [hunk 4]
```diff
@@ -0,0 +1,658 @@
...
+      <td style="text-align:left"></td>
+      <td style="text-align:left"></td>
+    </tr>
+    <tr>
+      <td style="text-align:left"></td>
+      <td style="text-align:left">AXONIQ-2000</td>
+      <td style="text-align:left">
+        <p><b>INVALID_SEQUENCE</b>
+        </p>
+        <p>This<b> </b>error is returned when there is a gap between the sequence
```
Comment (bert-laverman) [inline axon-server/administration/error-codes.md:156]: A bold space?

File: axon-server/administration/error-codes.md [hunk 5]
```diff
@@ -0,0 +1,658 @@
...
+          sent to the Axon Server cluster.</p>
+      </td>
+    </tr>
+    <tr>
+      <td style="text-align:left"></td>
+      <td style="text-align:left">AXONIQ-2101</td>
+      <td style="text-align:left">
+        <p><b>NOT_RUNNING_IN_CLUSTER</b>
+        </p>
+        <p>This error<b> </b>is returned when Axon Server is not running in a cluster.</p>
```
Comment (bert-laverman) [inline axon-server/administration/error-codes.md:201]: A bold space?

File: axon-server/administration/error-codes.md [hunk 6]
```diff
@@ -0,0 +1,658 @@
...
+          is rejected.</p>
+      </td>
+    </tr>
+    <tr>
+      <td style="text-align:left"></td>
+      <td style="text-align:left">AXONIQ-2302</td>
+      <td style="text-align:left">
+        <p><b>CONTEXT_CREATION_NOT_ALLOWED</b>
+        </p>
+        <p>This<b> </b>error is<b> </b>returned when an instruction to create a context
```
Comment (bert-laverman) [inline axon-server/administration/error-codes.md:242]: Two bold spaces.

File: axon-server/administration/error-codes.md [hunk 7]
```diff
@@ -0,0 +1,658 @@
...
+          on an Axon Server cluster is rejected due to invalid permissions.</p>
+      </td>
+    </tr>
+    <tr>
+      <td style="text-align:left"></td>
+      <td style="text-align:left">AXONIQ-2303</td>
+      <td style="text-align:left">
+        <p><b>NOT_SUPPORTED_IN_DEVELOPMENT</b>
+        </p>
+        <p>This<b> </b>error is<b> </b>returned when an instruction valid only for
```
Comment (bert-laverman) [inline axon-server/administration/error-codes.md:252]: Two bold spaces.

File: axon-server/administration/error-codes.md [hunk 8]
```diff
@@ -0,0 +1,658 @@
...
+          non-development environments is attempted on a development environment.</p>
```
Comment (bert-laverman) [inline axon-server/administration/error-codes.md:253]: "non-development mode is attempted when development mode is active."

File: axon-server/administration/error-codes.md [hunk 9]
```diff
@@ -0,0 +1,658 @@
...
+        <p>This error is returned when the maximum size of the cluster is reached.</p>
+      </td>
+    </tr>
+    <tr>
+      <td style="text-align:left"></td>
+      <td style="text-align:left">AXONIQ-2306</td>
+      <td style="text-align:left">
+        <p><b>ALREADY_MEMBER_OF_CLUSTER</b>
+        </p>
+        <p>This error is returned when<b> </b>the maximum size of the cluster is
```
Comment (bert-laverman) [inline axon-server/administration/error-codes.md:281]: Bold space.

File: axon-server/administration/error-codes.md [hunk 10]
```diff
@@ -0,0 +1,658 @@
...
+      </td>
+    </tr>
+    <tr>
+      <td style="text-align:left"></td>
+      <td style="text-align:left">AXONIQ-2307</td>
+      <td style="text-align:left">
+        <p><b>NOT_A_MEMBER</b>
+        </p>
+        <p>This error is returned when<b> </b>the maximum size of the cluster is
+          reached.</p>
```
Comment (bert-laverman) [inline axon-server/administration/error-codes.md:292]: A cluster related instruction was sent to a node that is not a member of a cluster.

File: axon-server/administration/error-codes.md [hunk 11]
```diff
@@ -0,0 +1,658 @@
...
+        </p>
+        <p>This error is<b> </b>returned when a node tries to join the cluster with
+          the same internal hostname and port as the leader.</p>
+      </td>
+    </tr>
+    <tr>
+      <td style="text-align:left"></td>
+      <td style="text-align:left">AXONIQ-2501</td>
+      <td style="text-align:left"><b>UNKNOWN_HOST</b>
+      </td>
```
Comment (bert-laverman) [inline axon-server/administration/error-codes.md:342]: A hostname was passed that did not resolve to a valid IP address.

**Review by bert-laverman — CHANGES_REQUESTED** [review]
Please do a global replace to remove the occurrences of "`<b> </b>`". (bold spaces)

File: axon-server/administration/error-codes.md (hunk 4 above)
Comment (smcvb) [reply axon-server/administration/error-codes.md:156]: I've seen this in more location actually...guessing it's a typo.

File: axon-server/administration/error-codes.md [hunk 12]
```diff
@@ -0,0 +1,658 @@
...
+          is too large.</p>
+      </td>
+    </tr>
+    <tr>
+      <td style="text-align:left"></td>
+      <td style="text-align:left">AXONIQ-2002</td>
+      <td style="text-align:left">
+        <p><b>TOO_MANY_EVENTS </b>
+        </p>
+        <p>This error is<b> </b>returned when an ad-hoc query (e.g. from the Search
```
Comment (smcvb) [inline axon-server/administration/error-codes.md:178]: Here is another bold space for example.

File: axon-server/administration/error-codes.md [hunk 13]
```diff
@@ -0,0 +1,658 @@
...
+          as it might be undergoing an update.</p>
+      </td>
+    </tr>
+    <tr>
+      <td style="text-align:left"></td>
+      <td style="text-align:left">AXONIQ-2200</td>
+      <td style="text-align:left">
+        <p><b>INVALID_TRANSACTION_TOKEN </b>
+        </p>
+        <p>This error is<b> </b>returned when during replication there is a mismatch
```
Comment (smcvb) [inline axon-server/administration/error-codes.md:220]: Bold space.

File: axon-server/administration/error-codes.md [hunk 14]
```diff
@@ -0,0 +1,658 @@
...
+          of the instruction passed to it.</p>
+      </td>
+    </tr>
+    <tr>
+      <td style="text-align:left"></td>
+      <td style="text-align:left">AXONIQ-1100</td>
+      <td style="text-align:left">
+        <p><b>NODE_IS_REPLICA</b>
+        </p>
+        <p>This error is<b> </b>returned when a request from a node to join a cluster
```
Comment (smcvb) [inline axon-server/administration/error-codes.md:78]: Bold space

File: axon-server/administration/error-codes.md [hunk 15]
```diff
@@ -0,0 +1,658 @@
...
+          the specified context.</p>
+      </td>
+    </tr>
+    <tr>
+      <td style="text-align:left"></td>
+      <td style="text-align:left">AXONIQ-1500</td>
+      <td style="text-align:left">
+        <p><b>AXONSERVER_NODE_NOT_CONNECTED</b>
+        </p>
+        <p>This<b> </b>error is returned when there is an instruction to create a
```
Comment (smcvb) [inline axon-server/administration/error-codes.md:140]: Bold space

File: axon-server/administration/error-codes.md (hunk 4 above)
Comment (smcvb) [reply axon-server/administration/error-codes.md:156]: I've seen this in more location actually...guessing it's a typo.

File: axon-server/administration/error-codes.md (hunk 12 above)
Comment (smcvb) [inline axon-server/administration/error-codes.md:178]: Here is another bold space for example.

File: axon-server/administration/error-codes.md [hunk 16]
```diff
@@ -0,0 +1,658 @@
...
+          inconsistency in the data between the nodes.</p>
+      </td>
+    </tr>
+    <tr>
+      <td style="text-align:left"></td>
+      <td style="text-align:left">AXONIQ-2301</td>
+      <td style="text-align:left">
+        <p><b>CLUSTER_NOT_ALLOWED</b>
+        </p>
+        <p>This error is<b> </b>returned when an instruction to create a cluster
```
Comment (smcvb) [inline axon-server/administration/error-codes.md:232]: Bold space

File: axon-server/administration/error-codes.md [hunk 17]
```diff
@@ -0,0 +1,658 @@
...
+          reached.</p>
+      </td>
+    </tr>
+    <tr>
+      <td style="text-align:left"></td>
+      <td style="text-align:left">AXONIQ-2307</td>
+      <td style="text-align:left">
+        <p><b>NOT_A_MEMBER</b>
+        </p>
+        <p>This error is returned when<b> </b>the maximum size of the cluster is
```
Comment (smcvb) [inline axon-server/administration/error-codes.md:291]: Bold space

File: axon-server/administration/error-codes.md [hunk 18]
```diff
@@ -0,0 +1,658 @@
...
+          reached.</p>
+      </td>
+    </tr>
+    <tr>
+      <td style="text-align:left"></td>
+      <td style="text-align:left">AXONIQ-2308</td>
+      <td style="text-align:left">
+        <p><b>INVALID_CONTEXT_NAME</b>
+        </p>
+        <p>This error is returned when<b> </b>an instruction is attemped on an invalid
```
Comment (smcvb) [inline axon-server/administration/error-codes.md:301]: Bold space

File: axon-server/administration/error-codes.md [hunk 19]
```diff
@@ -0,0 +1,658 @@
...
+          context.</p>
+      </td>
+    </tr>
+    <tr>
+      <td style="text-align:left"></td>
+      <td style="text-align:left">AXONIQ-2310</td>
+      <td style="text-align:left">
+        <p><b>CANNOT_REMOVE_LAST_NODE </b>
+        </p>
+        <p>This error is<b> </b>returned when a user tries to remove the last node
```
Comment (smcvb) [inline axon-server/administration/error-codes.md:311]: Bold space

File: axon-server/administration/error-codes.md (hunk 18 above)
Comment (smcvb) [inline axon-server/administration/error-codes.md:301]: I'd rephrase this slightly to `...an instruction is attempted on a non-existing context.`. Invalid feels ambiguous to me.

File: axon-server/administration/error-codes.md [hunk 20]
```diff
@@ -0,0 +1,658 @@
...
+          group.</p>
+      </td>
+    </tr>
+    <tr>
+      <td style="text-align:left"></td>
+      <td style="text-align:left">AXONIQ-2311</td>
+      <td style="text-align:left">
+        <p><b>INVALID_PROPERTY_VALUE</b>
+        </p>
+        <p>This error is returned in multiple conditions e.g.<b> </b>when creating
```
Comment (smcvb) [inline axon-server/administration/error-codes.md:322]: Bold space

File: axon-server/administration/error-codes.md [hunk 21]
```diff
@@ -0,0 +1,658 @@
...
+          the environment variable.</p>
+      </td>
+    </tr>
+    <tr>
+      <td style="text-align:left"></td>
+      <td style="text-align:left">AXONIQ-2500</td>
+      <td style="text-align:left">
+        <p><b>SAME_NODE_NAME </b>
+        </p>
+        <p>This error is<b> </b>returned when a node tries to join the cluster with
```
Comment (smcvb) [inline axon-server/administration/error-codes.md:334]: Bold space

File: axon-server/administration/error-codes.md [hunk 22]
```diff
@@ -0,0 +1,658 @@
...
+          is an error.</p>
+      </td>
+    </tr>
+    <tr>
+      <td style="text-align:left"></td>
+      <td style="text-align:left">AXONIQ-2510</td>
+      <td style="text-align:left">
+        <p><b>UNKNOWN_ROLE </b>
+        </p>
+        <p>This error is<b> </b>returned when an unknown role is assigned to a user
```
Comment (smcvb) [inline axon-server/administration/error-codes.md:360]: Bold space

File: axon-server/administration/error-codes.md [hunk 23]
```diff
@@ -0,0 +1,658 @@
...
+          interface directly or through the CLI.</p>
+      </td>
+    </tr>
+    <tr>
+      <td style="text-align:left"></td>
+      <td style="text-align:left">AXONIQ-2511</td>
+      <td style="text-align:left">
+        <p><b>INVALID_QUERY</b>
+        </p>
+        <p>This error is returned when<b> </b>the user send a query in the search
```
Comment (smcvb) [inline axon-server/administration/error-codes.md:371]: Bold space

File: axon-server/administration/error-codes.md [hunk 24]
```diff
@@ -0,0 +1,658 @@
...
+          Axon Server and there is an error while processing it.</p>
+      </td>
+    </tr>
+    <tr>
+      <td style="text-align:left"></td>
+      <td style="text-align:left">AXONIQ-4003</td>
+      <td style="text-align:left">
+        <p><b>COMMAND_DISPATCH_ERROR </b>
+        </p>
+        <p>This error is returned when a command messagae instruction is sent to
```
Comment (smcvb) [inline axon-server/administration/error-codes.md:433]:
```suggestion
        <p>This error is returned when a command message instruction is sent to
```

File: axon-server/administration/error-codes.md [hunk 25]
```diff
@@ -0,0 +1,658 @@
...
+      <td style="text-align:left"></td>
+      <td style="text-align:left"></td>
+    </tr>
+    <tr>
+      <td style="text-align:left"></td>
+      <td style="text-align:left">AXONIQ-7000</td>
+      <td style="text-align:left">
+        <p><b>NODE_NOT_READY_FOR_BACKUP</b>
+        </p>
+        <p>This<b> </b>error is returned when an Axon Server node is not available
```
Comment (smcvb) [inline axon-server/administration/error-codes.md:475]: Bold space

File: axon-server/administration/error-codes.md [hunk 26]
```diff
@@ -0,0 +1,658 @@
...
+          files.</p>
+      </td>
+    </tr>
+    <tr>
+      <td style="text-align:left"></td>
+      <td style="text-align:left">AXONIQ-9100</td>
+      <td style="text-align:left">
+        <p><b>DATAFILE_WRITE_ERROR </b>
+        </p>
+        <p>This error<b> </b>is returned when Axon Server is unable to write to the
```
Comment (smcvb) [inline axon-server/administration/error-codes.md:511]: Bold space

File: axon-server/administration/error-codes.md [hunk 27]
```diff
@@ -0,0 +1,658 @@
...
+          Event/Snapshot Data files.</p>
+      </td>
+    </tr>
+    <tr>
+      <td style="text-align:left"></td>
+      <td style="text-align:left">AXONIQ-9101</td>
+      <td style="text-align:left">
+        <p><b>INDEX_WRITE_ERROR </b>
+        </p>
+        <p>This error<b> </b>is returned when Axon Server is unable to write to the
```
Comment (smcvb) [inline axon-server/administration/error-codes.md:521]: Bold space

File: axon-server/administration/error-codes.md [hunk 28]
```diff
@@ -0,0 +1,658 @@
...
+          Index files.</p>
+      </td>
+    </tr>
+    <tr>
+      <td style="text-align:left"></td>
+      <td style="text-align:left">AXONIQ-9102</td>
+      <td style="text-align:left">
+        <p><b>DIRECTORY_CREATION_FAILED </b>
+        </p>
+        <p>This error<b> </b>is returned when Axon Server is unable to create a directory
```
Comment (smcvb) [inline axon-server/administration/error-codes.md:531]: Bold space

File: axon-server/administration/error-codes.md [hunk 29]
```diff
@@ -0,0 +1,658 @@
...
+          returned when the validation fails.</p>
+      </td>
+    </tr>
+    <tr>
+      <td style="text-align:left"></td>
+      <td style="text-align:left">AXONIQ-9900</td>
+      <td style="text-align:left">
+        <p><b>TRANSACTION_ROLLED_BACK</b> 
+        </p>
+        <p>This error<b> </b>is returned when any transaction is rolled back.</p>
```
Comment (smcvb) [inline axon-server/administration/error-codes.md:552]: Bold space

File: axon-server/administration/error-codes.md [hunk 30]
```diff
@@ -0,0 +1,658 @@
...
+          for events to be written to the event store segment.</p>
+      </td>
+    </tr>
+    <tr>
+      <td style="text-align:left"></td>
+      <td style="text-align:left">AXONIQ-6000</td>
+      <td style="text-align:left">
+        <p><b>NO_EVENTSTORE</b>
+        </p>
+        <p>This error<b> </b>is returned when the Axon Server Event Store is not
```
Comment (smcvb) [inline axon-server/administration/error-codes.md:571]: Bold space

File: axon-server/administration/error-codes.md [hunk 31]
```diff
@@ -0,0 +1,658 @@
...
+          available to perform any instructions.</p>
+      </td>
+    </tr>
+    <tr>
+      <td style="text-align:left"></td>
+      <td style="text-align:left">AXONIQ-6001</td>
+      <td style="text-align:left">
+        <p><b>CLIENT_DISCONNECTED </b>
+        </p>
+        <p>This error<b> </b>is returned when an Axon Framework client application
```
Comment (smcvb) [inline axon-server/administration/error-codes.md:581]: Bold space

File: axon-server/administration/error-codes.md [hunk 32]
```diff
@@ -0,0 +1,658 @@
...
+          the Axon Server cluster fails.</p>
+      </td>
+    </tr>
+    <tr>
+      <td style="text-align:left"></td>
+      <td style="text-align:left">AXONIQ-10007</td>
+      <td style="text-align:left">
+        <p><b>UNCOMMITTED_TERM </b>
+        </p>
+        <p>This error is returned when<b> </b>a request to update the configuration
```
Comment (smcvb) [inline axon-server/administration/error-codes.md:617]: Bold space

File: axon-server/administration/error-codes.md [hunk 33]
```diff
@@ -0,0 +1,658 @@
...
+      <td style="text-align:left"></td>
+      <td style="text-align:left"></td>
+    </tr>
+    <tr>
+      <td style="text-align:left"></td>
+      <td style="text-align:left">AXONIQ-0001</td>
+      <td style="text-align:left">
+        <p><b>OTHER</b>
+        </p>
+        <p>Any other<b> </b>errors.</p>
```
Comment (smcvb) [inline axon-server/administration/error-codes.md:644]: Bold space

**Review by smcvb — CHANGES_REQUESTED** [review]
Requesting changes in a similar fashion as Bert. Noted one typo, one thing I guess can be adjusted, but mainly because of the rogue "bold spaces".
